<launch>
  <group ns="tb3g">
    <rosparam file="$(find turtlebot3_carrier_icra2017)/param/initial_pose.yaml" command="load" />
    <rosparam file="$(find turtlebot3_carrier_icra2017)/param/target_pose.yaml" command="load" />

    <!-- pose_initialization -->
    <node pkg="turtlebot3_carrier_icra2017" type="pose_initialization" name="pose_initialization" output="screen">
      <param name="machine_name" type="str" value="tb3g" />
    </node>
  </group>
</launch>
